thermo_nerf/thermal_nerf/thermal_nerf_model.py

```diff
@@ -329,7 +330,9 @@ def get_loss_dict(
         thermal_batch = batch["thermal"].to(self.device)
 
         if self.field.pass_thermal_gradients:
-            loss_dict["thermal"] = self.thermal_loss(outputs["thermal"], thermal_batch)
+            loss_dict["thermal"] = self.thermal_loss(
```

Comment:
Why is the loss duct still using a string instead of the enum or at least the string from the enum